packages/device_info/ios/Classes/DeviceInfoPlugin.m

```diff
@@ -25,6 +28,11 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
       @"localizedModel" : [device localizedModel],
       @"identifierForVendor" : [[device identifierForVendor] UUIDString],
       @"isPhysicalDevice" : [self isDevicePhysical],
+      @"utsname.sysname" : @(un.sysname),
+      @"utsname.nodename" : @(un.nodename),
+      @"utsname.release" : @(un.release),
+      @"utsname.version" : @(un.version),
+      @"utsname.machine" : @(un.machine),
```

Comment:
I'm not well-versed in our `MethodChannel` conventions, but I'd expect the idiomatic way to pass a nested object would be by using a nested dictionary?